src/zabapgit_gui_router.prog.abap

```diff
@@ -289,9 +296,10 @@ CLASS lcl_gui_router IMPLEMENTATION.
 
     CREATE OBJECT lo_page
       EXPORTING
-        iv_key    = lv_key
-        is_file   = ls_file
-        is_object = ls_object.
+        iv_key           = lv_key
+        is_file          = ls_file
+        is_object        = ls_object
+        iv_supress_stage = boolc( iv_prev_page = 'PAGE_STAGE' ).
```

Comment:
I think we need to look into if this special case can be avoided, but we can do that later, after this PR is merged